### DISCUSSION TITLE
`issues list` should have an option to use filters

### DISCUSSION THREAD

**Discussion by jaymzh** [ankitpokhrel/jira-cli#265] [Ideas] [answered]
I have several filters defined in Jira. I'd like to be able to do something like

```
jira issues list --filter "John's Tasks"
```

or whatever the filter name is. I don't think that's currently available, but if I missed it, I apologize for the noise.

**Comment by ankitpokhrel** [comment, marked as answer]
@jaymzh You can filter by assigned user, right?

```sh
jira issue list -a"Johns username or email"
```

**Edit:** We can use raw query as stated in the comments.

```sh
jira issue list -q"filter='Chef Newb issues'"
```

**Reply by jaymzh** [reply]
@ankitpokhrel - that was a random example. I have things like "Chef Newb issues" which I have a saved filter of which is effectively: not closed/resolved/done/etc, and has label chef and has label noob.

I can repro that on the command line every time but I have to have filters in Jira anyway, so I'd rather not have to reproduce every filter I have in Jira manually on the command line (I mean, I'd make aliases, but still, I'd have to keep them all in two places)

**Reply by ankitpokhrel** [reply]
I see. I haven't checked but I think raw query should do the trick in this case.

```sh
jira issue list -q"filter='Chef Newb issues'"
```

**Reply by jaymzh** [reply]
That does indeed work!! Thanks!

Maybe we can just throw that into the help.

**Reply by ankitpokhrel** [reply]
Yes, we can definitely add this to the help/readme. PRs are welcome :)
